.github/workflows/test.yml

```diff
@@ -48,10 +53,15 @@ jobs:
         with:
           version: 7
           run_install: true
+      - id: get-options
+        run: . ./set-node-options.sh --action
```

Comment:
are we running out of memory or something?  shouldn't this not contribute to that issue?  or is it the `pixi.js` dependency?

Reply:
will that be removed once we migrate off the fork? it seems like it wasn't in that list of problematic deps

Reply:
I don't think it will be able to be removed after migrating off of the fork unless the next version of higlass has a smaller bundle size

Reply:
Ok, I understand that.  I am a little confused though - does that PR that we are waiting on not give `esm` code?  It seems to use `esbuild`, but I guess it could use `CommonJS`.

Reply:
Then yes it is possible that it will allow us to avoid bumping the memory limit. I have not looked into the specifics of the PR beyond the removal of the github dependencies in the higlass `package.json`

Reply:
My understanding is that there are bugs that need to be fixed in higlass, making the timeline for the next release uncertain, so I think we still need a different solution in the meantime

Reply:
Ok it can't hurt.  We'll see what happens next then.